Tutorials/Sprite-Related-Tutorials/Getting Started With Sprites in Splashkit Tutorial - C#/Getting Started With Sprites in Splashkit Tutorial - CSharp.mdx

```diff
@@ -105,21 +165,32 @@ is because we still have to do a basic
 [Window Refresh](https://splashkit.io/api/windows/#refresh-window) to 'update' or 'paint' what
 Sprites and where we have told SplashKit to put them. After you add these lines to our Main() :
 
+<Tabs>
+  <TabItem label="C#">
 ```csharp
 spriteWindow.Refresh(60);
 SplashKit.Delay(5000);
 ```
-
+  </TabItem>
+  <TabItem label="C++">
+```c++
+  refresh_screen();
```

Comment:
We might want to pass in the target fps, like so: `refresh_screen(60);` just so this matches with the C#.